### PR TITLE
feat: add `structuregraph-helpers`

### PR DESCRIPTION
**Pull request by kjappelbaum**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kjappelbaum** [comment]
@conda-forge/help-python, ready for review!

File: recipes/structuregraph-helpers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/structuregraph-helpers/meta.yaml:21]:
```suggestion
    - python >=3.8
```
Ref: https://github.com/kjappelbaum/structuregraph-helpers/blob/1d84458490a0be8d6685400283a5fe9dc7a0d0bc/setup.cfg#L58

File: recipes/structuregraph-helpers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/structuregraph-helpers/meta.yaml:23]:
```suggestion
    - python >=3.8
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

File: recipes/structuregraph-helpers/meta.yaml (hunk 1 above)
Comment (kjappelbaum) [reply recipes/structuregraph-helpers/meta.yaml:21]: thanks, It is actually an issue upstream in `setup.cfg` that I updated there now.

**Pull request merged**